admin-dev/themes/new-theme/js/pages/order/view/order-prices-refresher.js

```diff
@@ -37,7 +37,7 @@ export default class OrderPricesRefresher {
     $.ajax(this.router.generate('admin_orders_get_prices', {orderId}))
       .then((response) => {
         $(OrderViewPageMap.orderTotal).text(response.orderTotalFormatted);
-        $(OrderViewPageMap.orderDiscountsTotal).text(response.discountsAmountFormatted);
+        $(OrderViewPageMap.orderDiscountsTotal).text(`-${response.discountsAmountFormatted}`);
```

Comment:
This should be managed in the query that returns `discountsAmountFormatted` because CLDR manages some negative pattern for some languages
This could be done on the front side but since it's already a "formatted" value it should be easier to manage this on server side

Reply:
Ok I can change it

Reply:
We also have a cldr js library in the BO to do the job :wink: